action_tutorials/action_tutorials_cpp/README.md

```diff
@@ -0,0 +1,64 @@
+# Action Server
+
+In the constructor for `FibonacciActionServer`, an action server is created with callbacks that are called when a goal is received, when the goal is cancelled, and when the goal is accepted:
+
+```cpp
+this->action_server_ = rclcpp_action::create_server<Fibonacci>(
+  ...
+  "fibonacci",
+  std::bind(&FibonacciActionServer::handle_goal, this, _1, _2),
+  std::bind(&FibonacciActionServer::handle_cancel, this, _1),
+  std::bind(&FibonacciActionServer::handle_accepted, this, _1));
+```
+
+The `handle_goal` callback is called whenever a goal is sent to the action server by an action client. In the example code, the goal is accepted as long as the order is less than 46, otherwise it is rejected. This is to prevent potential integer overflow:
```

Comment:
Maybe it'd be nice to have a link to somewhere on the internet that talks about integer overflow.